docs/_plugins/dataviz/leaflet-l7.md

```diff
@@ -0,0 +1,11 @@
+---
+name: '@antv/l7-leaflet'
+category: dataviz
+repo: https://github.com/antvis/L7-Leaflet
+author: lzxue
+author-url: https://github.com/lzxue
+demo: http://l7-leaflet.antv.vision/leaflet_l7_layer
+compatible-v0:
```

Comment:
@Falke-Design does this need to be set explicitly?

Reply:
```suggestion
compatible-v0: false
```

Reply:
No not explicitly but I often make a commit and overwrite it with `false`.

BTW: should we introduce then compatible-v2?

Reply:
I think we can wait with `v2` until we stabilized things.